### DISCUSSION TITLE
Keycloak Redirect to Angular App

### DISCUSSION THREAD

**Discussion by sumalw** [keycloak/keycloak#24474] [Q&A]
In Keycloak after a successful login, is it not possible to redirect into the Angular route which enables router guard by the angular application? 

Ex:- Redirect URL which is protected by angular route guard= http://localhost:4200/dashboard  

Once authentication is successful, Keycloak will redirect to this URL. However, because of this guard, Keycloak not redirect properly and not receiving tokens.

**Comment by Bhavik237** [comment]
you can add  -  frame-src 'self'; frame-ancestors 'self' http://localhost:4200/dashboard; object-src 'none';   to realm settings ->security defences -> Content-Security-Policy .

Try this

**Reply by sumalw** [reply]
Hi [Bhavik237](https://github.com/Bhavik237),

Thank you for your quick reply. I added the same as you mentioned but got the same result. I've attached the image for you to look over.
![Screenshot 2023-11-02 133737](https://github.com/keycloak/keycloak/assets/10771887/7fa55280-8cda-46bb-aabd-113d5d2e8363)
